src/main/java/Duke.java

```diff
@@ -1,10 +1,531 @@
+    public static final String SPARKLY_EMOTICON =
+            "\u0028\uff89\u25d5\u30ee\u25d5\u0029\uff89\u002a\u003a\uff65\uff9f\u2727";
+    public static final String CONFUSED_EMOTICON = "\u0028\u30fb\u2227\u2010\u0029\u309e";
+    public static final String HAPPY_EMOTICON = "\uff08\u00b4\u30fb\u03c9\u30fb \u0060\uff09";
+    public static final String FLOWER_SMILE_EMOTICON = "(\u25e0\u203f\u25e0\u273f)";
+
+    public static final String SPARKLY_TEXT_DIVIDER = "\u2500\u2500\u2500\u2500\u2500\u2500\u2500 "
+            + "\u2731\u002a\u002e\uff61\u003a\uff61" + "\u2731\u002a\u002e\u003a\uff61\u2727\u002a\u002e\uff61"
+            + "\u2730\u002a\u002e\u003a\uff61\u2727" + "\u002a\u002e\uff61\u003a\uff61\u002a\u002e\uff61\u2731"
+            + " " + "\u2500\u2500\u2500\u2500\u2500" + "\u2500\u2500";
+
+    public static final int MAX_TASKS = 100;
+    public static final String TASKNAME_MARKER = "t/";
+    public static final String STARTTIME_MARKER = "s/";
+    public static final String ENDTIME_MARKER = "e/";
+    public static final String DUEDATE_MARKER = "d/";
+    public static final int NUM_LOGOS_AVALIABLE = 4;
+
+    // array containing all tasks the user has input
+    private static ArrayList<Task> allTasks = new ArrayList<Task>(MAX_TASKS);
+
+    private static int numTasks = 0;
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String userInput;
+        CommandType commandType = CommandType.START;
+
+        printTootieLogo();
+        printHelloMessage();
+
+        while(commandType != CommandType.BYE){
+            userInput = getUserInput();
+            echoUserInput(userInput);
+            printDivider();
+            commandType = extractCommandType(userInput);
+            executeCommand(commandType, userInput, allTasks);
+            printDivider();
+        }
+    }
+
+    // get user input, ignore comments and blank lines
+    private static String getUserInput() {
+        String userInput;
+        do {
+            userInput = SCANNER.nextLine();
+        } while (userInput.matches("(\r\n|[\n\r\u2028\u2029\u0085])?")
+                || userInput.startsWith(INPUT_COMMENT_MARKER));
+        return userInput;
+    }
+
+    // echo the userInput for testing
+    private static void echoUserInput(String userInput) {
+        System.out.println(userInput);
+        return;
+    }
+
+    // prints Tootie logo (text art randomized each run)
+    public static void printTootieLogo() {
+        String[] logos = new String[NUM_LOGOS_AVALIABLE];
+        logos[0] = SIMPLE_TOOTIE_LOGO;
+        logos[1] = BLOCKY_TOOTIE_LOGO;
+        logos[2] = TRAIN_THEME_TOOTIE_LOGO;
+        logos[3] = THICK_TOOTIE_LOGO;
```

Comment:
Are these strings going to be modified during runtime? If not then you can consider changing this array of String to a constant.